extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/StaticResourcesProcessor.java

```diff
@@ -82,6 +86,31 @@ public void nativeImageResource(Optional<StaticResourcesBuildItem> staticResourc
         }
     }
 
+    //    @BuildStep
+    public void processGeneratedResources(
+            List<GeneratedStaticResourceBuildItem> generatedStaticResourceBuildItems,
+            BuildProducer<GeneratedResourceBuildItem> generatedResourceBuildItemBuildProducer,
+            BuildProducer<NativeImageResourceBuildItem> nativeImageResourcesProducer,
+            BuildProducer<AdditionalStaticResourceBuildItem> additionalStaticResourceBuildItemBuildProducer,
```

Comment:
Only for prod mode, we have two options:
1. we could use `additionalStaticResourceBuildItemBuildProducer`
2. we could create a new StaticHandler

I think `additionalStaticResourceBuildItemBuildProducer` seems better.